matchmaking.py

```diff
@@ -14,18 +14,20 @@ def __init__(self, li, config, username):
         self.last_challenge_created = time.time()
         self.last_game_ended = time.time()
         self.challenge_expire_time = 25  # The challenge expires 20 seconds after creating it.
+        self.challenge_timeout = max(self.matchmaking_cfg.get("challenge_timeout") or 30, 1)

```

Comment:
Convert `challenge_timeout` to seconds here so everything in `should_create_challenge()` is in seconds.